lib/charms/kafka/v0/kafka_snap.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""## Overview
+
+`kafka_snap.py` provides a collection of common functions for managing snap installation and
+config parsing common to both the Kafka and ZooKeeper charms
+
+"""
+
+import logging
+from typing import Dict, List
+
+from ops.model import StatusBase
+from charms.operator_libs_linux.v0 import apt
+from charms.operator_libs_linux.v1 import snap
+from ops.model import ActiveStatus, BlockedStatus, MaintenanceStatus
```

Comment:
Remove previous sole import of `StatusBase` and add it to this list
```suggestion
from ops.model import ActiveStatus, BlockedStatus, MaintenanceStatus, StatusBase
```